packages/core/src/constants.ts

```diff
@@ -10,6 +10,15 @@ export const SEQUENCE_HASH_LENGTH = 7;
 /** @internal */
 export const SEQUENCE_PREFIX = '___';
 
+/** @internal */
+export const DEBUG_SEQUENCE_SEPARATOR = '_';
+
+/** @internal */
+export const SEQUENCE_SIZE =
+  process.env.NODE_ENV === 'production'
```

Comment:
To enable debug, we should check, in addition to NODE_ENV, a local storage flag.
See https://github.com/microsoft/fluentui/blob/f9ca45d012d2e77eea728e19651b144452562004/packages/fluentui/styles/src/debugEnabled.ts for inspiration

Reply:
We should check how it works with SSR and you can't get `localStorage` from the server :)

Reply:
I applied the localStorage solution like the example from Miro. 
But IDK how to handle it with SSR. Will check or shout for help on Monday

Reply:
Updated it to use debug hash always for non prod :)